app/scripts/modules/kubernetes/src/v2/pipelines/stages/deleteManifest/deleteManifestStage.ts

```diff
@@ -1,28 +1,20 @@
-import { module } from 'angular';
-
-import { Registry, SETTINGS, IStage, ExecutionDetailsTasks } from '@spinnaker/core';
+import { Registry, IStage, ExecutionDetailsTasks } from '@spinnaker/core';
 
 import { manifestExecutionDetails } from 'kubernetes/v2/pipelines/stages/ManifestExecutionDetails';
 import { manifestSelectorValidators } from '../validators/manifestSelectorValidators';
 import { DeleteManifestStageConfig } from 'kubernetes/v2/pipelines/stages/deleteManifest/DeleteManifestStageConfig';
 
-export const KUBERNETES_DELETE_MANIFEST_STAGE = 'spinnaker.kubernetes.v2.pipeline.stage.deleteManifestStage';
-
 const STAGE_NAME = 'Delete (Manifest)';
 const STAGE_KEY = 'deleteManifest';
-module(KUBERNETES_DELETE_MANIFEST_STAGE, []).config(() => {
-  // Todo: replace feature flag with proper versioned provider mechanism once available.
-  if (SETTINGS.feature.versionedProviders) {
```

Comment:
I don't think we need to use this anymore -- we're explicitly excluding v2 stage types from v1 (and vice-versa) when registering each version of the provider with an `unsupportedStageTypes` list.